R/normalize.R

```diff
@@ -245,6 +245,9 @@ normalize_store_arg <- function(store, storage_options=NA, mode=NA) {
   }
 
   if(is.character(store)) {
+    if(grepl("^https?://", store)) {
+      return(HttpStore$new(store))
+    }
```

Comment:
needed to explicitly check for http / https to get HttpStores to open with convenience functions